x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/flash_messages_logic.ts

```diff
@@ -17,10 +18,13 @@ export interface IFlashMessage {
 
 export interface IFlashMessagesLogicValues {
```

Comment:
To be consistent with other components, shouldn't this be `IFlashMessagesValues` and `IFlashMessagesActions`? I see this has been done in other places but I question the value of adding `Logic` to the interface names, which we have never done until recently.

Reply:
I think this was a convention we started doing in App Search in our [Kea HOC->hook tech debt conversion](https://github.com/elastic/app-search-team/issues/884).

I know App Search has this convention in almost all its logic files, but I'm guessing Workplace Search is different? I have absolutely 0 strong feelings one way or another, I'm good with whatever we decide on as a team

Reply:
@byronhulcher @JasonStoltz do either of you have a compelling reason to include `Logic` in the interface names? Just curious why the decision was made. We don't do it in other places like `IEngineOverviewComponentProps`. It's clear by "Props" that this is a component and I submit that using "Values" and "Actions" gives context that these are from a logic file.

Reply:
I don't care one way or the other.

Reply:
I like the more verbose `IFlashMessagesLogicValues` form but I'm not stuck to it

Reply:
Looks like we have 1 for without Logic, 1 for with, and 2 abstains - anyone have any preferences on how to tiebreak? 🤪 Maybe we can get @yakhinvadim in on this?

Reply:
Lol, I also don't have a preferred way :D

Reply:
Vadim voted a "fine either way" so I'll go ahead and reluctantly tiebreak here and say we can probably lose `Logic` without much context loss. I'll go ahead and make the change now - hope that sounds good to everybody!

Reply:
Addressed in adc814e